src/desktop/apps/artist/routes.coffee

```diff
@@ -35,6 +35,10 @@ sd = require('sharify').data
 
       return res.redirect(artist.href) unless(_.find nav.sections(), slug: tab) or artist.counts.artworks is 0
 
+      # TODO: ARTIST_MARKET_DATA_TEST remove after test closes
+      labFeatureGate = req.user && req.user.hasLabFeature('Artist Market Data Summary')
+      testGroup = if labFeatureGate then res.locals.sd.ARTIST_MARKET_DATA_TEST else 'control'
```

Comment:
Here is where testGroup will be overwritten for now unless lab feature is enabled

Reply:
Ah, ok- yea this makes more sense. I was initially surprised to see anything in `running_tests`, since thats usually part of the 'last' PR to swap out gate for actual test.

But I see how you're really trying to minimize those last-minute swap outs, which I think is a great call! Very nice 👏 

This seems cool, if you have the lab feature then you can check the various states via the usual `?split_test[artist_market_data_test]=...` query param.

Reply:
Yep - the query param work once the lab feature is set up here. Otherwise those are ignored(overwritten with control view rendering)